### PR TITLE
[RW-929][risk=no] Review column changes

### PR DESCRIPTION
**Pull request by freemabd1**
added new columns to all review tabs.

### CODE REVIEW

File: api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh [hunk 1]
```diff
@@ -91,8 +91,8 @@ SELECT P.PERSON_ID,
 	c2.CONCEPT_NAME as SOURCE_NAME,
 	c2.CONCEPT_CODE as SOURCE_CODE,
 	c2.VOCABULARY_ID as SOURCE_VOCABULARY,
-	t.VISIT_OCCURRENCE_ID as VISIT_ID,
-	 v.visit_concept_id,
+	CASE WHEN t.VISIT_OCCURRENCE_ID is null then 0 else t.VISIT_OCCURRENCE_ID end as VISIT_ID,
+	CASE WHEN v.visit_concept_id is null then 0 else v.visit_concept_id end as visit_concept_id,
```
Comment (osgsusan) [inline api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh]: We may want to find a better way to do this when we have time.

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java [hunk 2]
```diff
@@ -13,92 +12,160 @@
...
+      "source_name as sourceName,\n" +
+      "source_code as sourceCode,\n" +
+      "source_vocabulary as sourceVocabulary,\n" +
+      "age_at_event as ageAtEvent,\n" +
+      "visit_type as visitType,\n" +
+      "source_value as sourceValue,\n" +
+      "num_mentions as numMentions,\n" +
+      "first_mention as firstMention,\n" +
+      "last_mention as lastMention\n";
+
```
Comment (dolbeew) [inline api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: This looks like a good place to refactor column names

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java [hunk 3]
```diff
@@ -13,92 +12,160 @@
...
+      case CONDITION:
+        return CONDITION_SQL_TEMPLATE;
+      case PROCEDURE:
+        return PROCEDURE_SQL_TEMPLATE;
+      case OBSERVATION:
+        return OBSERVATION_SQL_TEMPLATE;
+      case MEASUREMENT:
+        return MEASUREMENT_SQL_TEMPLATE;
+      default:
+        return PHYSICAL_MEASURE_SQL_TEMPLATE;
```
Comment (osgsusan) [inline api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Should the default be physical measurements?

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java [hunk 4]
```diff
@@ -13,92 +12,160 @@
...
-          .setNamedParameters(params)
-          .setUseLegacySql(false)
-          .build();
-    }
+  private static final String BASE_SQL_TEMPLATE =
+    "select person_id as personId,\n" +
+      "data_id as dataId,\n" +
+      "start_datetime as startDate,\n";
+
+  private static final String ALL_EVENTS_SQL_TEMPLATE =
```
Comment (osgsusan) [inline api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Maybe refactor these sql templates later to reduce some duplication.

File: api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh (hunk 1 above)
Comment (freemabd1) [reply api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh]: Agreed, will have a followup PR that can address this.

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java (hunk 4 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Yep.

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java (hunk 2 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: Will do in another PR.

File: api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java (hunk 3 above)
Comment (freemabd1) [reply api/src/main/java/org/pmiops/workbench/cohortreview/ReviewTabQueryBuilder.java]: We can rethink this.

File: api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh (hunk 1 above)
Comment (osgsusan) [reply api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh]: Sounds good.

**Pull request merged**